src/plugins/dataflow-tool/dataflow-logger.ts

```diff
@@ -0,0 +1,89 @@
+import { LogEventName, Logger, DataflowProgramChange } from "../../lib/logger";
+import { Connection, Control, Node } from "rete";
+
+// I would like to use Record<string, any> instead of "object," but need to implement correctly
```

Comment:
Or, as I know was mentioned, leave the type as is or as `any` since this data is destined for outside the application.

Reply:
In the future you can make the DataflowProgramChange a record, and give it a key that represents what it is - whether Node or anything else.